repair/repair.cc

```diff
@@ -154,7 +154,11 @@ static std::vector<table_id> get_table_ids(const replica::database& db, const ss
     table_ids.reserve(tables.size());
     for (auto& table : tables) {
         thread::maybe_yield();
-        table_ids.push_back(db.find_uuid(keyspace, table));
+        try {
+            table_ids.push_back(db.find_uuid(keyspace, table));
+        } catch (replica::no_such_column_family&) {
+            rlogger.warn("Column family {} does not exist in keyspace {}", table, keyspace);
```

Comment:
You can simply print the current_exception, it already contains the formatted string with the needed information